### DISCUSSION TITLE
Whisper trigger bug

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2685] [Q&A] [answered]
If i put a button that is **disabled** inside of Whisper which triggers on hover, it is not triggering because of disabled state.

If i put a button inside of Badge, Whisper again is not triggering on hover or click.

**Comment by simonguo** [comment, marked as answer]
Event listeners on disabled elements are disabled, you can refer to this example.
https://rsuitejs.com/components/tooltip/#disabled-elements
